src/components/CarouselCard/styled/indicatorButton.js

```diff
@@ -35,19 +41,19 @@ const StyledIndicatorButton = styled.button`
 
     :focus {
         outline: 0;
-        box-shadow: ${SHADOW_OUTLINE};
-        border: 1px solid ${COLOR_BRAND_ACTIVE};
+        box-shadow: ${props => props.brandShadow};
+        border: 1px solid ${props => props.brandMainColor};
     }
 
     ${props =>
         props.isSelected &&
         `
-            background: ${COLOR_BRAND};
-            border: 1px solid ${COLOR_BRAND};
+            background: ${props.brandMainColor};
+            border: 1px solid ${props.brandMainColor};
 
             :hover {
-                background: ${COLOR_BRAND_ACTIVE};
-                border: 1px solid ${COLOR_BRAND_ACTIVE};
+                background: ${props.successMainColor};
+                border: 1px solid ${props.successMainColor};
```

Comment:
@TahimiLeonBravo  check this styles

Reply:
@yvmunayev we should pass the `brandDarkColor` when `:hover`